solidity-v1/dashboard/src/css/withdraw-granted-tokens.less

```diff
@@ -0,0 +1,101 @@
+.withdraw-granted-tokens {
+  &__form_checkbox {
```

Comment:
Unnecessary class.

Reply:
Fixed in b3c8dc6dd9dca8fcaa8b49daa5d3ff13a8b4e84a